src/auth/authentication.js

```diff
@@ -7,6 +7,13 @@ export function isLoggedIn() {
     return true;
 }
 
+export function checkUser(userId) {
+    if (userId == getUserId()) {
```

Comment:
Is there a difference between `==` and `===` for this?

Reply:
I think `==` does a type conversion, if there is no need we could do `===`

Reply:
Oh true, I'll switch that over